scripts/background.js

```diff
@@ -3,7 +3,7 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
   if (
     changeInfo.status === "complete" &&
     tabUrl &&
-    tabUrl.includes("someurl.com")
+    tabUrl.includes("app.slack.com")
```

Comment:
This is the url for Slack workspaces on the web.

Reply:
This browser extension only injects the above CSS when "app.slack.com" is in the URL